src/App.tsx

```diff
@@ -1,61 +1,186 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './Types/user';
+import { TodoList } from './components/TodoList';
+import { Todo } from './Types/todo';
+import { FormFields } from './Types/formFields';
+import { findUserByUserId } from './utils/fundUserByUserId';
+import { ErrorsFields } from './Types/errorsFields';
+
+const formFields: FormFields = {
+  title: '',
+  selectedUserId: '0',
+};
+
+const handleAddNewTodo = (
+  id: number,
+  title: string,
+  userId: number,
+  user: User | null,
+  completed: boolean = false,
+) => {
+  return {
+    id: id,
+    title: title.trim(),
```

Comment:
Only what this function does trim the title and it doesn't add todo, so the name is incorrect